### PR TITLE
Translated a bunch of stuff and updated to 0910

### PR DESCRIPTION
**Pull request by Milky00000**
Updating to 0910 seems to have removed a bunch of uslesse parts, hence why wall of stuff being removed. Also adding parts to be able to load in recollection.

By the way the CommonEvents.json seems to be broken somewhere. Don't know if you ever took a look but you can't even open the menu without it crashing. Formatting the file caused it to not be able to load the menu .img and probably other stuff. I wanted to take a look by the file is so huge...

### CODE REVIEW

File: www/data/Map030.json [hunk 1]
```diff
@@ -5307,14 +5366,14 @@
                             "code": 401,
                             "indent": 0,
                             "parameters": [
-                                "\\n< Serena >(It seems like something I shouldn't be\nlooking at...!)"
+                                "\\n< Serena >(Like something you're not supposed to see...!)"
                             ]
                         },
                         {
-                            "code": 0,
```
Comment (dazedanon) [inline www/data/Map030.json:5314]: Keep at 0

File: www/data/Map030.json [hunk 2]
```diff
@@ -5307,14 +5366,14 @@
...
+                            "code": 401,
                             "indent": 0,
                             "parameters": [
-                                ""
+                                "\\n<セレナ>"
```
Comment (dazedanon) [inline www/data/Map030.json:5376]: TL This

File: www/data/Map046.json [hunk 3]
```diff
@@ -4020,7 +4048,7 @@
                             "code": 401,
                             "indent": 3,
                             "parameters": [
-                                "\\n< Serena >(The Goblin Cave...! Again, in a situation\nlike that...!?)"
+                                "\\n< Serena >!Goblin cave...!\nIt's like that again...! ? )"
```
Comment (dazedanon) [inline www/data/Map046.json:4051]: Fix this sentence

File: www/data/Map046.json [hunk 4]
```diff
@@ -14109,14 +14137,14 @@
                             "code": 401,
                             "indent": 0,
                             "parameters": [
-                                "\\n< Serena >.........Let's go back. We need to go back and clean\nup......."
+                                "………戻ろう。戻って、綺麗にしないと……。"
```
Comment (dazedanon) [inline www/data/Map046.json:14140]: Fix this

File: www/data/Map046.json [hunk 5]
```diff
@@ -14109,14 +14137,14 @@
...
                             ]
                         },
                         {
-                            "code": 0,
+                            "code": 401,
```
Comment (dazedanon) [inline www/data/Map046.json:14144]: Keep at 0

**Review by dazedanon — CHANGES_REQUESTED** [review]
Couple of things that need fixes. Also the menu doesn't crash at all for me. Unsure why you are seeing that on latest.\

Also can you update the system.json with the right version number when you get the chance?

**Comment by Milky00000** [comment]
Trying to figure how all this work,. I put a bunch of updates but it created new commits instead of updating. I'll do a new pull request when i fixed all my stuff.

**Pull request closed without merging**